src/ex1/script.js

```diff
@@ -0,0 +1,52 @@
+let pendingTodos = [];
+
+function addTodo() {
+  let currentTodo = document.querySelector(".todoInput").value;
+
+  if (currentTodo === "") {
+    alert("Add a value to create a new todo");
+  } else if (pendingTodos.includes(currentTodo)) {
```

Comment:
a nice workaround for the not unique tasks problem, think of a different approach that would solve it without restricting the user from adding tasks with the same name